runtime/src/asset.rs

```diff
@@ -1475,9 +1539,19 @@ impl<T: Trait> Module<T> {
         Self::_update_checkpoint(ticker, to_did, current_to_balance);
 
         <BalanceOf<T>>::insert(&ticker_to_did, updated_to_balance);
-        <Tokens<T>>::insert(ticker, token);
-
-        Self::deposit_event(RawEvent::Issued(ticker.clone(), to_did, value));
+        <Tokens<T>>::insert(ticker.clone(), token);
+        let round = Self::funding_round(ticker.clone());
```

Comment:
We can pass a reference here rather than a clone.`Self::funding_round(ticker.clone())` should work fine.